src/sentry/lang/javascript/processor.py

```diff
@@ -545,7 +545,11 @@ def preprocess_step(self, processing_task):
 
     def handles_frame(self, frame, stacktrace_info):
         platform = frame.get("platform") or self.data.get("platform")
-        return settings.SENTRY_SCRAPE_JAVASCRIPT_CONTEXT and platform in ("javascript", "node")
+        has_context_line = len(frame.get("context_line", "")) > 0
+        has_lineno = frame.get("lineno", -1) >= 0
```

Comment:
These can also all be `None`. would also do `bool(frame.get("lineno"))` etc.

Reply:
But `bool(0)` would result in `False` and having col/line `0` is valid right?

Reply:
So it seems to do what I want:
```
>>> frame = { "a": None }
>>> print frame.get("a", -1)
None
>>> print frame.get("a", -1) > 0
False
```

Reply:
The latter is undefined. It tries `None` > 0 which on Python 3 will be a `TypeError`.

```
>>> None > 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

Reply:
Saw my chance of teaching Mr. Python something and failed 😞 
Will fix it :)